.github/workflows/docstring_validation.yml

```diff
@@ -0,0 +1,50 @@
+name: Test Docstrings Validation
+
+on:
+  pull_request:
+    paths:
+      - "integration-tests/**"
+
+jobs:
+  betelgeuse:
+    name: "betelgeuse dry-run"
+    runs-on: ubuntu-latest
+    container:
+      image: fedora:latest
+
+    - uses: actions/checkout@v4
+
+    steps:
+    - name: Base setup for Betelgeuse
+      run: |
+        dnf --setopt install_weak_deps=False install -y \
+            python3-pip
+        python3 -m pip install betelgeuse
+        cp integration-tests/custom_betelgeuse_config.py \
+          /usr/local/lib/python3.*site-packages/
+
+    - name: Run Betelgeuse
+      run: |
+        betelgeuse --config-module \
+            custom_betelgeuse_config test-case --dry-run \
+            integration-tests/ dryrun_project ./test_case.xml
```

Comment:
since `betelgeuse` uses `importlib.import_module()` to import the config moduled passed via `--config-module`, it is enough to have the path of that module available in `PYTHONPATH` -- something like this:
```
PYTHONPATH=integration-tests/ betelgeuse --config-module custom_betelgeuse_config [etc...]
```